.prettierignore

```diff
@@ -1,3 +1,5 @@
+*.hbs
```

Comment:
I feel we should implement #1432 soonish. I also struggled with IDE being setup to run Prettier and seeing templates being changed. Configuring Prettier to ignore Glimmer templates seems to be a good interim solution. 👍